src/tools/CMakeLists.txt

```diff
@@ -7,7 +7,13 @@ set(rados_srcs
   ${PROJECT_SOURCE_DIR}/src/common/obj_bencher.cc
   ${PROJECT_SOURCE_DIR}/src/osd/ECUtil.cc)
 add_executable(rados ${rados_srcs})
-target_link_libraries(rados librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS} radosstriper)
+
+target_link_libraries(rados librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS})
+if(WITH_LIBRADOSSTRIPER)
+  target_link_libraries(rados radosstriper)
+else(WITH_LIBRADOSSTRIPER)
+  target_link_libraries(rados cls_lock_client)
```

Comment:
https://github.com/ceph/ceph/pull/21983#discussion_r189762038

Reply:
A bit of a quandry! Either omitting cls_lock_client or adding librados results in linker errors. I am building with "make rados", so perhaps it's somehow related to that, but so far I don't see how. 

I've corrected the formatting, thanks.